Dockerfile

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2020. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+###############################################################################
+FROM golang:1.17-alpine
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh docker
+
+WORKDIR /go/src/github.com/tap8stry/orion
+
+COPY pkg/    pkg/
+COPY cmd/    cmd/
+
+# Copy Go Modules manifests and download the modules
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+#RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build --tags static_all -v -o ./bin/orion cmd/api-mgr/main.go
+
+RUN wget https://download.clis.cloud.ibm.com/ibm-cloud-cli/2.4.0/binaries/IBM_Cloud_CLI_2.4.0_linux_amd64.tgz &&\
```

Comment:
Can we remove this ? This is specific to IBM registry and we can move this to prior step.

Reply:
removed